test/health_check_test_case.py

```diff
@@ -84,6 +89,35 @@ def test_health_endpoint_keys(self):
                         self.assertEqual(200, response.status_code)
                         self.assertEqual(expected_response, response.json())
 
+    @mock_s3
+    @mock_sts
+    @mock_sqs
+    def test_cached_health(self):
+        storage_service = StorageService()
+        storage_service.create_bucket()
+        with ResponsesHelper() as helper:
+            helper.add_passthru(self.base_url)
+            response = requests.get(self.base_url + '/health/cached')
+            self.assertEqual(500, response.status_code)
```

Comment:
If possible, this section should assert that the error is because the object is missing. That would help with clarifying the intent of the test, too. At the very least there should be a comment explaining each of the three sections of this test.

Reply:
I guess the primary error is not exposed.